dev/Deployment/DeploymentManager.cpp

```diff
@@ -46,21 +46,22 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
 
     winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentResult DeploymentManager::Initialize()
     {
-        ::WindowsAppRuntime::Deployment::Activity::Context::Get().GetActivity().Start(false,
-                                                                                      Security::IntegrityLevel::IsElevated(),
-                                                                                      AppModel::Identity::IsPackagedProcess(),
-                                                                                      Security::IntegrityLevel::GetIntegrityLevel());
+        ::WindowsAppRuntime::Deployment::Activity::Context::Get().SetActivity(WindowsAppRuntimeDeployment_TraceLogger::Initialize::Start(false,
```

Comment:
Multiple calls to `::WindowsAppRuntime::Deployment::Activity::Context::Get().blahblah`

SUGGESTION:
```
auto& activityContext{ ::WindowsAppRuntime::Deployment::Activity::Context::Get() };
activityContext.SetActivity(...)
...
activityContext.SetActivity(...)
```